src/api/levelpack.js

```diff
@@ -82,6 +84,30 @@ const getMultiRecords = async LevelPackName => {
   return times.filter(t => !t.Level.Hidden);
 };
 
+// for int total times only for now.
+const getIntBestTimes = async KuskiIndex => {
+  const IntLevelPackIndex = 84;
+
+  const q =
+    'SELECT * FROM legacybesttime WHERE KuskiIndex = ? ' +
```

Comment:
if levelpack.Legacy is 0 this needs to use table besttime and if it's 1 use table legacybesttime (this value is not the same on dev and live atm)

Reply:
It's only for the internal level pack atm since we don't currently need it for other level packs. So I think internal would always be legacy, or no ? Not sure.

Reply:
internals are not (yet) legacy on live. It'll be soon, so we can wait or you can change it :)